src/compiler/crystal/tools/doc/method.cr

```diff
@@ -175,4 +176,15 @@ class Crystal::Doc::Method
   def has_args?
     !@def.args.empty? || @def.block_arg || @def.yields
   end
+
+  def to_json(io)
+    {
+      name:     name,
+      args:     args_to_s,
+      doc:      doc,
+      prefix:   prefix,
```

Comment:
we probably want `kind` to be, in the JSON output, one of "instance", "class" or "top-level". The current `kind` and `prefix` mehods are just convenient for the HTML output.